Formula/arcs.rb

```diff
@@ -40,9 +41,17 @@ def install
       "--prefix=#{prefix}",
       "--with-boost=#{Formula["boost"].opt_include}"
     system "make", "install"
+    libexec_src = Pathname.new("#{libexec}/bin/src")
+    libexec_src.install "src/long-to-linked-pe"
```

Comment:
Is there both a binary executable `long-to-linked-pe` and a script also named `long-to-linked-pe`?